### PR TITLE
Contribute route

### PR DESCRIPTION
**Pull request by rfpoulos**
@ellenkramp , @ponchieponcho   please okay going into master!

### CODE REVIEW

File: html/contributeRoute.html [hunk 1]
```diff
@@ -7,6 +7,29 @@
     <title>Document</title>
 </head>
 <body>
+    <form action="/my-handling-form-page" method="post">
+          <div>
+            <label for="contributer_id">User ID:</label>
```
Comment (ponchieponcho) [inline html/contributeRoute.html]: Is this going to be a unique ID or just the username? Otherwise I would just make it userName

Comment (ponchieponcho) [reply html/contributeRoute.html]: Also I think they should just be users in general as they can be both a user and contributer

Comment (ponchieponcho) [reply html/contributeRoute.html]: @rfpoulos I'm doneeeee

Comment (ellenkramp) [reply html/contributeRoute.html]: @rfpoulos I'm done as well - looks like a great start!

**Comment by rfpoulos** [comment]
updated to record input into local array, mergeable? @ellenkramp @ponchieponcho

**Comment by rfpoulos** [comment]
@ellenkramp @ponchieponcho  pls confirm merge

**Comment by rfpoulos** [comment]
@ellenkramp  @ponchieponcho  hey guys, pls approve this to add!

**Pull request merged**